api/app/signals/settings/base.py

```diff
@@ -445,6 +444,9 @@
 
     # Enable/disable status mail for Handled after negative feedback
     'REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED': os.getenv('REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED', False) in TRUE_VALUES, # noqa
+
+    # Enable/disable only mail when Feedback allows_contact is True
+    'REPORTER_MAIL_DISABLE_CONTACT_FEEDBACK_ALLOWS_CONTACT': os.getenv('REPORTER_MAIL_DISABLE_CONTACT_FEEDBACK_ALLOWS_CONTACT', True) in TRUE_VALUES, # noqa
```

Comment:
Please rename to `REPORTER_MAIL_CONTACT_FEEDBACK_ALLOWS_CONTACT_ENABLED` so that the naming is inline with other feature flags like `REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED`